server/server.go

```diff
@@ -24,6 +24,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/open-policy-agent/opa/internal/json/patch"
```

Comment:
nit: move the import down to be the rest of the internal/OPA imports.

Reply:
Done